api/vendor/github.com/openshift/assisted-service/models/cluster_create_params.go

```diff
@@ -52,7 +52,7 @@ type ClusterCreateParams struct {
 	// Installation disks encryption mode and host roles to be applied.
 	DiskEncryption *DiskEncryption `json:"disk_encryption,omitempty" gorm:"embedded;embeddedPrefix:disk_encryption_"`
 
-	// Guaranteed availability of the installed cluster. 'Full' installs a Highly-Available cluster
+	// (DEPRECATED) please use 'control_plane_count' instead. Guaranteed availability of the installed cluster. 'Full' installs a Highly-Available cluster
```

Comment:
nit:
```suggestion
	// (DEPRECATED) Please use 'control_plane_count' instead. Guaranteed availability of the installed cluster. 'Full' installs a Highly-Available cluster
```
for consistency with other deprecation messages

Reply:
Same for all other spots

Reply:
Thanks !